Tom Tarpey: Intro Python day 1 (mvp) - working on the toy programs

decagondev opened the pull request:
done the hello world, bignum and datatypes to have a few code changes so that I could do an initial pull request going to work on the rest after lunch

decagondev commented:
done day 1 mvp complete and stretch for day 1

RockmanExe commented:
Everything looks great, keep it up!

decagondev commented:
i completed the MVP for today and done some of tomorrows work and a bit of freestyle random commands as a bit of stretch

RockmanExe commented:
Great job meeting mvp and completing the stretch so early! Game looks fun, nice!

decagondev commented:
done todays mvp and tomorrows mvp and some stretch. also made some stretch where the game can call native programs of the os if you have a laptop item and have it turned on in the game. currently implemented, Ping, clear and a web browser. will implement more along the way

RockmanExe commented:
Great job on all the stretch stuff, and the art!

decagondev commented:
got a few more classes made added another room and refactored a few bits. also added the skeleton of the win scenario based upon collecting certain items to get in to a final room. the item classes still need a little work to get the win functionality but the skeleton is complete to get it working. all in all. got mvp for all the days + some stretch and also some random made up stretch too

RockmanExe commented:
Great job this week, game looks amazing!
